ajax/libs/bootstrap-year-calendar/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "bootstrap-year-calendar",
+  "description": "A fully customizable year calendar widget, for Bootstrap!",
+  "homepage": "http://www.bootstrap-year-calendar.com/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Paul-DS/bootstrap-year-calendar"
+  },
+  "author": "Paul-DS <@Paul-DS> (https://taylorhakes.com/)",
+  "license": "Apache-2.0",
+  "filename": "bootstrap-year-calendar.js",
+  "keywords": [
+    "calendar",
+    "bootstrap"
+  ],
+  "npmName": "bootstrap-year-calendar",
+  "npmFileMap": [
+    {
+      "basePath": "js",
+      "files": [
+        "bootstrap-year-calendar*"
+      ]
+    },
+    {
+      "basePath": "css",
+      "files": [
+        "bootstrap-year-calendar*"
+      ]
+    }
```

Comment:
It's not good to use `js` and `css` as basePath. Let's leave basePath blank and use following as the rule:
```
    {
      "basePath": "",
      "files": [
        "+(js|css)/**/bootstrap-year-calendar*"
      ]
    }
```
Thanks.